clarify how to go to EAP menu (#6048)
MicrosoftDocs/windows-itpro-docs: windows/client-management/advanced-troubleshooting-802-authentication.md

 
![eap authentication type comparison](images/comparisontable.png)

> If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu (In **NPS snapp-in** go to **Policies** -> **Network Policies** -> Right click on the policy and select **Properties** -> In the appeared window go to **Constraints** tab and select **Authentication methods** section):

Reviewer: ```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select select the **Authentication methods** section.
```

> If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snap-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.

Reviewer: ```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snap-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication Methods** section.
```

Reviewer 2: @MaratMussabekov - please merge this additional commit. Thank you.

> If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.

Reviewer 3: Correction of minor typo ("snapp-in" `->` snap-in):
```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snap-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.
```

Reviewer 2: @MaratMussabekov - please merge this additional commit. Thank you.

> If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select select the **Authentication methods** section.

Reviewer: ```suggestion
If a certificate is used for its authentication method, check if the certificate is valid. For server (NPS) side, you can confirm what certificate is being used from the EAP property menu. In **NPS snapp-in**, go to **Policies** > **Network Policies**. Right click on the policy and select **Properties**. In the pop-up window, go to the **Constraints** tab and select the **Authentication methods** section.
```

Reviewer 2: @MaratMussabekov - please merge this additional commit. Thank you.
